### DISCUSSION TITLE
Idea / Feature Request: Presentation Mode for Shared Sessions

### DISCUSSION THREAD

**Discussion by oisteinhaugland** [excalidraw/excalidraw#8519] [General]
I had this a requested to me during a shared excalidraw session. 

It was a big canvas, with a lot of work on it, and we were going to review it together.

The person asked me if i could be the "guide" and take control of the "camera" so to speak. That wasn't possible, so we moved around individually.

Feature-request: Being able to guide the viewport of all members of a shared session, so they can follow your view.

---

Thoughts?
Is this something that would be a good addition?
I have no clue how hard it is to implement.

**Comment by Mrazator** [comment]
We have this built-in, it's called a follow mode!

https://x.com/excalidraw/status/1737130730587603053?lang=es
